### PR TITLE
Feature/aleste new1

### PR DESCRIPTION
**Pull request by alesteka**
Take a look

### CODE REVIEW

**Review by nkosi-tauro — CHANGES_REQUESTED** [review]
Make sure to register your model so that it appears in the Admin Panel. 
I have added some comments

File: user_service/views.py [hunk 1]
```diff
@@ -138,4 +140,67 @@ def login_service(request):
...
+            type = request.POST["type"]
+            severity = request.POST["severity"]
+            description = request.POST["description"]
+            FlawFormModel.objects.filter(id = flaw).update(type=type, severity = severity, description = description)
+            return redirect('employeeview', user_id)  # Redirect back to employee view
+        else:
+            context["not_valid"] = 1
+            return render(request, 'employeeview/edit_flaw.html', context)
+
+    return render(request, 'employeeview/edit_flaw.html', context)
```
[the remark this answers is not in the document]
Comment (alesteka) [reply user_service/views.py]: done, ran pylint and moved to appropriate folder

File: user_service/models.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+class FlawFormModel(models.Model):
+    '''
+    Reporting Form Model
+    '''
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    type = models.CharField(max_length=100, choices=Vulnerability_Type, blank=False)
+    severity = models.CharField(max_length=10, choices=CHOICES, blank=False,validators=[validate_choice])
+    description = models.CharField(max_length=200, blank=True)
+
+    def __str__(self):
```
[the remark this answers is not in the document]
Comment (alesteka) [reply user_service/models.py]: done

File: user_service/models.py
[the remark this answers is not in the document]
Comment (alesteka) [reply user_service/models.py:1]: moved

File: user_service/views.py [hunk 3]
```diff
@@ -138,4 +140,67 @@ def login_service(request):
...
+    '''
+    Service to modify the chosen flaw.
+    '''
+    form = AddFlawForm()
+    context = {"form": form, 
+               "user_id": user_id,
+               "flaw_id": flaw
+               }
+    if request.method == 'POST':
+        form = AddFlawForm(request.POST)
```
[the remark this answers is not in the document]
Comment (alesteka) [reply user_service/views.py]: done

File: user_service/models.py [hunk 4]
```diff
@@ -1,3 +1,57 @@
...
+        raise ValidationError('severity type not selected')
+
+class FlawFormModel(models.Model):
+    '''
+    Reporting Form Model
+    '''
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    type = models.CharField(max_length=100, choices=Vulnerability_Type, blank=False)
+    severity = models.CharField(max_length=10, choices=CHOICES, blank=False,validators=[validate_choice])
+    description = models.CharField(max_length=200, blank=True)
```
[the remark this answers is not in the document]
Comment (alesteka) [reply user_service/models.py]: changed - done

**Pull request merged**